overlore/llm/constants.py

```diff
@@ -82,15 +83,39 @@
 )
 
 AGENT_CREATION_TEMPLATE = """
-    {name}, a {sex} {role}, has just arrived in {realm_name} to settle.
-    Upon entering the realm, {name} addresses the lord with the following matters:
-
-    1. Expresses gratitude for the warm welcome and hospitality extended by the realm.
-    2. Highlights the skills and contributions he/she brings as a {role}.
-    3. Explain his background and where he comes from.
-    4. Discusses the significance of recent events: {events}.
-
-    During the speech, {name} maintains a demeanor that is {happiness}, {hunger} and {belligerent}.
+    Generate for an npc:
+     - age
+     - name
+     - surname
+     - {trait_type} character trait
+     - sex
+     - role
+     List of sex: {sex}
+     List of roles: {roles}
+"""
 
-    Format the speech like so: <name>:<speech>.
+AGENT_CREATION_EXAMPLE = """
```

Comment:
Let's name this `AGENT_CREATION_SYSTEM_PROMPT_TEMPLATE`. I think it should be 
```
Imagine you're the game master for a strategy game, you are tasked with creating Non-Playable Characters. Use these examples as reference: [enter examples]. 
Generate for an npc:
     - age
     - name
     - surname
     - character trait
     - sex (male or female)
     - role (miner, farmer, ...)
The user will tell you if the character trait needs to be positive or negative
```
I think you're mixing up what the user and system prompt needs to be. The system prompt should be the one defining what the task of the LLM is going to be. The user then uses that definition of an "assistant" to do its tasks.

Tell me how that version goes